lib/preload.js

```diff
@@ -682,9 +682,11 @@ class Preloader extends EventEmitter {
     .then((registryToken) => {
       const opts = { authconfig: { registrytoken: registryToken } }
       // Docker connection
+      // We use localhost on windows because of this bug in old node versions:
```

Comment:
Would probably spell out that node version <8.10.0, but that's just cherry on the top :)